scalableminds/webknossos
# Create a layer for each channel of NGFF-Zarr datasets

### URL of deployed dev instance (used for testing):
- ~~https://channelindex.webknossos.xyz/~~

### Steps to test:
- Import https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.4/idr0062A/6001240.zarr via remote zarr import
- Should result in two layers, one for each channel
- give data loading a minute or two, then data should load (use wk layer coloring in left sidebar to distinguish the channels)

Not tested: Importing zarr datasets with only one channel. This should result in only one layer.

### Issues:
- fixes #6578 

------
(Please delete unneeded items, merge only when none are left open)
